src/Altinn.Notifications.Email.Core/Altinn.Notifications.Email.Core.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.22.0" />
```

Comment:
not a fan of this event grid dependency on core, but I guess there is not better way to implement this?

Reply:
I agree, but we are just using the models to simplify event-handling from eventgrid.